sdk/turing/router/config/router_ensembler_config.py

```diff
@@ -47,7 +50,7 @@ def type(self) -> str:
 
     @type.setter
     def type(self, type: str):
-        assert type in {'standard', 'docker'}
+        assert type in {'standard', 'docker', "pyfunc"}
```

Comment:
Super nit: We should be consistent the use of `''` vs `""` in python. I generally prefer the [black](https://github.com/psf/black) code formatter which picks double quotes over single but that's debatable.